frontend/src/redux/auth/reducer.js

```diff
@@ -52,6 +52,7 @@ const authLoginFailReducer = (state, action) => {
 const authLogoutReducer = (state, action) => {
   localStorage.removeItem("token");
   localStorage.removeItem("user");
+  localStorage.removeItem("userId");
```

Comment:
I noticed both `user` and `userId` were used in different parts of the project. I wanted to include both for dependency purposes for now.